bin/epaa.py

```diff
@@ -934,7 +938,11 @@ def make_predictions_from_peptides(peptides, methods, tool_thresholds, alleles,
         pred_dataframes.append(df)
 
     # write prediction statistics
-    statistics = {'prediction_methods': [ method + "-" + version for method, version in methods.items() ],'number_of_variants': '-', 'number_of_unique_peptides': [str(p) for p in peptides], 'number_of_unique_peptides_after_filtering': [str(p) for p in peptides_filtered]}
+    statistics = {'prediction_methods': [ method + "-" + version for method, version in methods.items() ],
+                'number_of_variants': 0,
```

Comment:
@christopher-mohr does it also make sense to you here to update number of variants to zero rather than `"-"` as otherwise they cannot be added up in the `merge_json.py` file.

Reply:
Sure, it makes sense. 👍 I guess I introduced it before splitting of peptide input was possible.

Reply:
perfect, then everything works as expected, I'll merge the PR